src/environments/environment.ts

```diff
@@ -67,3 +67,9 @@ export const decorateModuleRef = _decorateModuleRef;
 export const ENV_PROVIDERS = [
   ...PROVIDERS
 ];
+
+export const environment = {
+  production: false,
+
+  apiUrl: 'http://localhost:3000'
```

Comment:
This seems unused.

Reply:
It is, I'll revert changes to environment.ts